packages/ui-react/src/components/TextField/TextField.tsx

```diff
@@ -70,7 +70,7 @@ const TextField: React.FC<Props> = ({
           {isTextArea(inputProps) ? (
             <textarea id={id} className={styles.input} rows={3} ref={textAreaRef} {...inputProps} />
           ) : (
-            <input id={id} className={styles.input} type={'text'} ref={inputRef} {...inputProps} />
+            <input id={id} className={styles.input} type={'text'} aria-required={inputProps.required} ref={inputRef} {...inputProps} />
```

Comment:
`aria-required` VS `required` we don't need both, right? 🤔

Reply:
Yes, I see what you mean. But when looking at the HTML structure of this `TextField` for example, I don't see any indication for accessibility users to see that it is required, or am I missing something? 

![Screenshot 2024-01-24 at 17 43 23](https://github.com/Videodock/ott-web-app/assets/48496458/85ee46d0-c740-4579-981e-4c7336417804)

Reply:
I think we should only add `aria-required` on non-semantic elements, so not on inputs (provided that `required` is set):

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-required#:~:text=When%20a%20semantic,ARIA%20role%20assigned.